storage/sms.php

```diff
@@ -44,7 +44,7 @@ function template3($cc_name, $cc_phonenumber, $mother_name, $mother_number){
 	}
 	
 	
-	function template4($phone_number, $cc_name, $phone_number, $mother_name, $mother_number){
+	function template4( $cc_name, $phone_number, $mother_name, $mother_number){
```

Comment:
sms code was crashing earlier 
see this
![screenshot 95](https://cloud.githubusercontent.com/assets/20840850/17591927/df590158-5ffd-11e6-9558-5ef69007dc96.png)
![screenshot 95](https://cloud.githubusercontent.com/assets/20840850/17591933/e26ba472-5ffd-11e6-8999-24a1c82b443b.png)